src/views/GovernanceReviewTeam/index.tsx

```diff
@@ -56,101 +70,106 @@ const GovernanceReviewTeam = () => {
     });
 
   return (
-    <PageWrapper className="easi-grt">
-      <Header />
-      <MainContent>
-        <section className="easi-grt__request-summary">
-          <div className="grid-container padding-y-2">
-            <BreadcrumbNav>
-              <li>
-                <Link className="text-white" to="/">
-                  Home
-                </Link>
-                <i className="fa fa-angle-right margin-x-05" aria-hidden />
-              </li>
-              <li>{systemIntake.requestName}</li>
-            </BreadcrumbNav>
-            <dl className="easi-grt__request-info">
-              <div>
-                <dt>{t('intake:fields.projectName')}</dt>
-                <dd>{systemIntake.requestName}</dd>
-              </div>
-              <div className="easi-grt__request-info-col">
-                <div className="easi-grt__description-group">
-                  <dt>{t('intake:fields.requester')}</dt>
-                  <dd>{requesterNameAndComponent}</dd>
+    userGroupsSet &&
```

Comment:
Don't render anything until we know we can see this content. It looks like a large diff, but it's mostly indentation.